x/fungible/genesis.go

```diff
@@ -18,14 +18,12 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 		k.SetSystemContract(ctx, *genState.SystemContract)
 	}
 
-	k.SetParams(ctx, genState.Params)
```

Comment:
Do we plan to clean up this `Params` from chain state thru migration script or forget about it?